### PR TITLE
Issue #1756: Added data to datepicker for preselecting pending time

### PR DESCRIPTION
**Pull request by stefanhaerter**
Closing #1756

### CODE REVIEW

**Review by svenoe — COMMENTED** [review]
Please set DiffTime to 0 if old until time was present.
Please use % for PendingTimeSettings.
Please add SysConfig-Setting "Remember last pending time".
Please check whether RealTimeTillNotUsed is later than now.

**Comment by stefanhaerter** [comment]
RealTillTimeNotUsed only appears within the code I added within this PR in the whole file.

Note for Development: Check whether RestorePendingInformation retrieves some relevant information from walking through the history.

File: Kernel/Config/Files/XML/Ticket.xml [hunk 1]
```diff
@@ -4256,6 +4256,13 @@
             <Item ValueType="String" ValueRegex="">%%Pending</Item>
         </Value>
     </Setting>
+    <Setting Name="Ticket::Frontend::AgentTicketActionCommon###RememberLastPendingTime" Required="0" Valid="1">
```
Comment (svenoe) [inline Kernel/Config/Files/XML/Ticket.xml]: Ticket::Frontend::RestorePendingTime

File: Kernel/Config/Files/XML/Ticket.xml [hunk 2]
```diff
@@ -4256,6 +4256,13 @@
...
+        <Description Translatable="1">Defines if in input masks based on AgentTicketActionCommon a potentially set pending time should be prefilled if it is later than now.</Description>
+        <Navigation>Frontend::Agent::View::TicketActionCommon</Navigation>
```
Comment (svenoe) [inline Kernel/Config/Files/XML/Ticket.xml]: see PendingDiffTime

File: Kernel/Config/Files/XML/Ticket.xml [hunk 3]
```diff
@@ -4256,6 +4256,13 @@
             <Item ValueType="String" ValueRegex="">%%Pending</Item>
         </Value>
     </Setting>
+    <Setting Name="Ticket::Frontend::AgentTicketActionCommon###RememberLastPendingTime" Required="0" Valid="1">
+        <Description Translatable="1">Defines if in input masks based on AgentTicketActionCommon a potentially set pending time should be prefilled if it is later than now.</Description>
```
Comment (svenoe) [inline Kernel/Config/Files/XML/Ticket.xml]: Restore last pending time in input masks based on AgentTicketActionCommon, if pending state is set.

**Review by svenoe — CHANGES_REQUESTED** [review]

**Comment by stefanhaerter** [comment]
See https://github.com/RotherOSS/otobo/issues/1756#issuecomment-1750572649

**Pull request merged**